### ISSUE TITLE
How to change the loss function （MSE loss-->GIoU loss）?I tried，but when I train it again，I failed。。Please help me！！！Thanks！

### ISSUE THREAD

**Issue by xiaoyuzhu666** [eriklindernoren/PyTorch-YOLOv3#484]

**Comment by 948024326** [comment]
Do you solve it? I want to do this too....

**Comment by Flova** [comment]
This part of the code has been updated or removed. I, therefore, close this issue now. Feel free to reopen for further discussion.
